antlr-kotlin-examples-js/src/test/kotlin/MiscStringTest.kt

```diff
@@ -11,19 +11,19 @@ fun assertArrayEquals(a: CharArray, b: CharArray) {
 class MiscStringTest {
 
     @Test fun testToCharArrayEmpty() {
-        assertArrayEquals(charArrayOf(), "".toCharArray())
+        assertArrayEquals(charArrayOf(), "".map { it }.toCharArray())
```

Comment:
Why this change? It is repeated a lot of times but I do not understand why

Reply:
The String.toCharArray is currently experimental feature in kotlin common stdlib, but already there. Until they remove the experimental state of it, I think this is cleaner then providing the same method in the library, just to avoid collisions in naming.

Reply:
I would suggest we instead create an helper method (that we can also inline), so that we avoid repeating this pattern over and over. I think it would make for more readable code.

For example:

```
inline fun String.asCharArray() = this.map { it }.toCharArray()

...
assertArrayEquals(charArrayOf(), "".asCharArray())
...

```

Reply:
I can agree on that